generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -4,8 +4,8 @@
 import <%=packageName%>.<%= mainClass %>;
 import <%=packageName%>.domain.<%= entityClass %>;
 import <%=packageName%>.repository.<%= entityClass %>Repository;<% if (service != 'no') { %>
-import <%=packageName%>.service.<%= entityClass %>Service;<% } else { if (searchEngine == 'elasticsearch') { %>
```

Comment:
why did you remove the else statement?  when there is service layer tests should be triggered via that instead of repository